crates/opencascade-sys/cpp/wrapper.cpp

```diff
@@ -10,6 +10,10 @@ rust::String type_name(const HandleStandardType& handle) {
   return std::string(handle->Name());
 }
 
+std::unique_ptr<gp_Pnt> HandleGeomCurve_Value(const HandleGeomCurve& curve, const Standard_Real U) {
+  return make_unique<gp_Pnt>(curve->Value(U));
```

Comment:
Sorry to beat a dead horse, but is there a particular reason you need `make_unique` here vs. just using `return std::unique_ptr<T>(...);` as it exists elsewhere in the code?

I don't code in C++ in my day-to-day so maybe I'm missing something.

Reply:
```suggestion
  return std::unique_ptr<gp_Pnt>(new gp_Pnt(curve->Value(U)));
```

We should be able to use it in this way without needing to create the `make_unique` util function. Sorry if it seems old fashioned, but I'd just prefer all the wrapper code to follow the same pattern so we can change them all at once later with less confusion on why one function used `make_unique` and another used the `unique_ptr` constructor directly.

Reply:
Updated